modules/t/hgvs_parser.t

```diff
@@ -698,6 +698,15 @@ sub test_output{
   ok(  $hgvs_genomic->{$allele} eq $output->[0], "$input genomic level $output->[0], $output->[5]" );
   if($DEBUG==1){print "TEMP: $input => gen; expected $output->[0]\t returns: $hgvs_genomic->{$allele} for allele:$allele\n";} 
 
+  # hgvs genomic with slice and without a slice 
```

Comment:
This test gets done several times. It should be run outside this sub and only once.